sirepo/auth/ldap.py

```diff
@@ -40,9 +40,12 @@ def _bind(creds):
                 m = "Unable to contact LDAP server"
                 if isinstance(e, ldap.INVALID_CREDENTIALS):
                     m = "Incorrect email or password"
-                else:
-                    pkdlog("email={} exception={}", creds.email, e)
-                raise sirepo.util.Response(self.reply_ok(PKDict(formError=m)))
+                raise sirepo.util.UserAlert(
```

Comment:
Use `raise sirepo.util.SReplyExc(self.reply_ok(PKDict(formError=m)))`

SReplyExc replaced Response, because Response and Reply had very different meanings.